tests/test_virtual_graph_composite.py

```diff
@@ -74,7 +74,7 @@ def test_embedding_parameter(self):
         sampler = self.sampler
 
         __, __, adj = sampler.structure
-        embedding = {v: {v} for v in adj}
```

Comment:
Is this change required to make it work? Or just an aesthetic change?

Reply:
EmbeddingStructure normalizes the embedding's values to be tuples.  The impacted context is actually a few lines down, line 81:

```
        self.assertEqual(sampler.embedding, embedding)
```